package com.harshu.execption;
//exception with try and catch
public class Division {

	public static void divide() {
		int a=6, b=0,c;
		try {
			c=a/b;
			System.out.println("division"+c);
					
		}
		catch(Arithemetic)
	}
}
